src/main/java/seedu/address/ui/ModuleCard.java

```diff
@@ -33,28 +33,32 @@
     @FXML
     private Label id;
     @FXML
-    private Label phone;
-    @FXML
-    private Label address;
+    private Label nextLesson;
     @FXML
-    private Label email;
+    private Label nextExam;
     @FXML
     private FlowPane tags;
 
     /**
-     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     * Creates a {@code ModuleCode} with the given {@code Module} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex) {
+    public ModuleCard(Module module, int displayedIndex) {
         super(FXML);
-        this.person = person;
+        this.module = module;
+        String modName;
+        String modCode = module.getCode().toString();
+
+        if (module.getName().isPresent()) {
+            modName = module.getName().get().toString();
+        } else {
+            modName = "";
+        }
+
+        String moduleHeader = String.format("%s %s", modCode, modName);
```

Comment:
i think might be better to declare header as just `modCode` first. then if module.getName.isPresent() then u can append a `" modName"` to the header